ziti/edge/network/factory.go

```diff
@@ -128,6 +128,21 @@ func (conn *routerConn) NewDialConn(service *rest_model.ServiceDetail) *edgeConn
 	return edgeCh
 }
 
```

Comment:
Would it make sense to have a container message if you're sending multiple responses?

Reply:
I actually do have it already. I just failed to use it.